.claude/skills/preflight/references/ci-comment-template.md

```
@@ -0,0 +1,180 @@
+
+```diff
+- <old code>
++ <new code>
+```
+
+</details>
+
+<details>
+<summary>🤖 Prompt for AI Agents</summary>
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)